composer.json

```diff
@@ -17,11 +17,11 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.1",
-        "silverstripe/cms": "^4.4@dev",
-        "silverstripe/admin": "^1.4@dev",
-        "silverstripe/versioned": "^1.3@dev",
-        "silverstripe/versioned-admin": "^1.2@dev",
-        "symbiote/silverstripe-gridfieldextensions": "^3.1",
+        "silverstripe/cms": "^4.6@dev",
+        "silverstripe/admin": "^1.6@dev",
+        "silverstripe/versioned": "^1.6@dev",
+        "silverstripe/versioned-admin": "^1.6@dev",
+        "symbiote/silverstripe-gridfieldextensions": "^3.2",
```

Comment:
```suggestion
        "silverstripe/cms": "^4.4@dev",
        "silverstripe/admin": "^1.4@dev",
        "silverstripe/versioned": "^1.3@dev",
        "silverstripe/versioned-admin": "^1.2@dev",
        "symbiote/silverstripe-gridfieldextensions": "^3.1",
```

Reply:
Build was failing because some matrix couldn't resolve to an installable set of packages.